appveyor.yml

```diff
@@ -0,0 +1,35 @@
+---
+
+init:
+  - set PATH=C:\Python27\Scripts;%PATH%             # while python's bin is already in PATH, but pip.exe in Scripts\ dir isn't
+  - set PATH=C:\msys64\mingw64\bin;C:\msys64\usr\bin;%PATH%
+
+environment:
+  DEGREE_OF_PARALLELISM: 3
+  matrix:
+  - GENERATOR: MSYS Makefiles
+    CONFIG: Release
+    PARALLEL_FLAG: -j
+  - GENERATOR: Visual Studio 14 2015
```

Comment:
Agreed but I haven't tried that on AppVeyor yet. We might want to land it this way first (since we know it works) and then try to add 2013.